update.go

```diff
@@ -363,6 +363,9 @@ other options are ignored.
 				}
 				config.IntelRdt = &configs.IntelRdt{}
 				intelRdtManager := intelrdt.NewManager(&config, container.ID(), state.IntelRdtPath)
+				if intelRdtManager == nil {
```

Comment:
Thanks your contribution.
I think the only position of returning `nil` is here:
https://github.com/opencontainers/runc/blob/314b79aeca1bad2a850e92b85907564f98ec8060/libcontainer/intelrdt/intelrdt.go#L162-L165

But unfortunately, we didn't return the error to the caller, so I think we should add an error log in L163 to let the user know what's happened. WDYT?